habitat/datasets/image_nav/__init__.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from habitat.core.dataset import Dataset
+from habitat.core.registry import registry
+
+
+# TODO(akadian): This is a result of moving SimulatorActions away from core
```

Comment:
Pretty sure Kadian won't like being assigned TODOs now :-).

Reply:
🙂 copied from objectnav init. these are littered across all dataset inits and could stand to be cleaned up.